assets/app/features/upgrade/templates/landing-page.less

```diff
@@ -11,7 +11,10 @@
     }
```

Comment:
Not your changes, but could you please move the `.h4` styles from line 13 inside this `h1`?
```
h1 {
    ...

    &.h4 {
        display: inline-block;
        margin-bottom: 27px;
        padding-bottom: 15px;
        padding-left: 15px;
     }
}

```